spdx/writers/jsonyamlxml.py

```diff
@@ -413,6 +414,9 @@ def create_snippet_info(self, annotations_by_spdx_id):
                 snippet_object["licenseInfoFromSnippet"] = list(
                     map(self.license, snippet.licenses_in_snippet)
                 )
+            byte_range = {"endPointer": {"offset": snippet.byte_range[1], "reference": snippet_object["fileId"]},
```

Comment:
Sometimes you're using the `snippet_from_file_spdx_id` local variable, and sometimes reading the same value via `snippet_object["fileId"]`. Both options seem fine to me (slight preference to the former, since we don't need to specify the "fileid" every time), but you shuold be consistent